rules/TypeDeclaration/Rector/ClassMethod/AddParamTypeBasedOnPHPUnitDataProviderRector.php

```diff
@@ -85,6 +93,11 @@ public function test(string $value)
     {
     }
 
+    #[\PHPUnit\Framework\Attributes\DataProvider('provideData')]
+    public function test_with_attribute(string $name)
+    {
+    }
```

Comment:
Feel free to keep only one example, so docs are short to read and point is passed. Feel free to pick one you prefere

Reply:
Thanks, removed the commit I added so it is the original code example for now